INSTALL.md

```diff
@@ -108,7 +124,18 @@ $ make
 
 The executable will be located on the `bin` directory.
 
-### Installing
+#### Windows
+
+Run CMake to generate a solution in visual studio (note: modify the command below to reflect your vcpkg installation folder).
+
+```bash
+$ cmake -DCMAKE_TOOLCHAIN_FILE=D:\\vcpkg\\scripts\\buildsystems\\vcpkg.cmake ..
+```
+
+Then, open the solution file (`grSim.sln`), and build project grSim. Make sure to set the project to compile the `Release` version and not the `Debug` version.
```

Comment:
Then, open the solution file (`grsim.sln`) in Visual Studio, and build project grSim. Make sure to set the project to compile in `Release` version instead of `Debug` version.